custom_components/openwrt/services.yaml

```diff
@@ -27,4 +27,24 @@ exec:
       description: Arbitrary object added to the execute result event
       required: false
       selector:
-        object: {}
\ No newline at end of file
+        object: {}
+init:
+  name: Managing services
+  target:
+    device:
+      integration: openwrt
+  fields:
+    name:
+      name: Name
+      description: Service Name
+      required: true
+      example: "dnsmasq"
+      selector:
+        text: {}
+    action:
```

Comment:
The PR looks good! thanks for the contribution.

One suggestion, the `action` field could be of type `select`, as described here: https://www.home-assistant.io/docs/blueprint/selectors/#select-selector

Reply:
Thanks, good idea.